src/LoginPage.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Paper, Grid, TextField, Button } from "@material-ui/core";
+import { useFormik } from "formik";
+
+const validateLogin = (loginData) => {
+  const errors = {};
+
+  if (!loginData.email) {
+    errors.email = "Please Enter Email";
+  } else if (
+    !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(loginData.email)
+  ) {
+    errors.email = "Invalid email address";
+  }
+
+  if (!loginData.password || loginData.password.length === 0) {
+    errors.password = "Please Enter Password";
+  } else if (loginData.password.length < 6) {
+    errors.password = "Password must contain at least 6 characters";
```

Comment:
can you add this validation also to password, It is long, contains uppercase letters, lowercase letters, numbers, and special characters

Reply:
https://github.com/RahafSalah/second-react-app/pull/6